experiment/Commonfunctions/commonfunctions-demo.js

```diff
@@ -293,14 +299,17 @@ function posttorder(root){
 
 }
 
-function recursivePostorder(root) {
+export function recursivePostorder(root) {
     ans=[];
     posttorder(root);
-    console.log(ans);
+
     dem=setInterval(demo,2000);
 }
 
 function reset(){
    document.getElementById("startNext").innerHTML="Start";
    resetTraversal();
 }
+window.reset = reset;
```

Comment:
So, the only functions that are made global are the ones that have an effect on the site itself. Since I ran into many errors while modularizing code, this was one of them. I needed to make the functions that corresponded to the buttons global so that both the modularization and the functionality was preserved. I might be exporting some functions that I don't need to be exporting, I will take a look at it .